src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Jobs/StopAzureRmRecoveryServicesBackupJob.cs

```diff
@@ -24,18 +24,27 @@
 
 namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets
 {
+    /// <summary>
+    /// Cancels a job. Returns the corresponding job object after this operation finishes.
```

Comment:
Cancels a job. Returns the corresponding job object after the trigger of the cancellation finishes.  The job may not cancel successfully. The cmdlet will ensure that the service is notified that a cancellation has been triggered.